src/runtime/runtime.js

```diff
@@ -474,6 +474,11 @@ export class Runtime {
   getLogger() {
     return this.configured_(true).then((runtime) => runtime.getLogger());
   }
+
+  /** @override */
+  eventManager() {
+    return this.configured_(true).then((runtime) => runtime.eventManager());
```

Comment:
Isn't this returning a promise to an event manager?

Reply:
It's kind of odd to change the name also isn't it?

Reply:
Not sure what you mean by changing the name?

Reply:
getEventManager vs eventManager

Reply:
There is also an existing eventManager function in ConfiguredRuntime that returns event manager directly.

Reply:
ah, yes I named them differently because eventManager returns the event manager directly but getEventManager returns a Promise.

Reply:
So what is this new function overriding then?  Should it be called getEventManager and override that?  Sorry if I'm slightly confused